.github/workflows/release_mac.yml

```diff
@@ -106,7 +106,7 @@ jobs:
         pytest
 
     - name: Verify ONNX with ort-nightly
-      if: matrix.python-version != '3.6' # Do not test ort-nightly for Python 3.6 due to missing (unsupported) package
+      if: matrix.python-version != '3.10' # FIXME Do not test ort-nightly for Python 3.10 due to missing (unsupported) package
```

Comment:
What's the FIXME? Is there a reason 3.10 cannot be tested here? Thanks!

Reply:
Sorry for the confusion. The reason is ort-nightly has not supported Python 3.10 yet: https://test.pypi.org/project/ort-nightly/#files. I think I should use TODO instead, which is more proper. Just updated. Please review it again. Thank you!